src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -152,18 +175,16 @@ public void run() {
         while (iter.hasNext()) {
           SourceDocument d = iter.next();
           if (d == null) {
-            LOG.info("Null document encountered");
+            counters.unindexedDocuments.incrementAndGet();
             continue;
           }
           if (!d.indexable()) {
-            LOG.info("Document is not indexable");
             counters.unindexableDocuments.incrementAndGet();
             continue;
           }
 
           @SuppressWarnings("unchecked") // Yes, we know what we're doing here.
           Document doc = transformer.createDocument(d);
-
           if (doc != null && (whitelistDocids == null || whitelistDocids.contains(d.id()))) {
             if (args.uniqueDocid) {
               writer.updateDocument(new Term("id", d.id()), doc);
```

Comment:
What about the `else` case? Should we increment `counters.unindexedDocuments` here?